src/ui/lib/components/deployment-actions.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts"></script>
+
+<div class="deployment-actions" aria-label="action buttons">
```

Comment:
When should `div` vs `Box` be used?

Reply:
I typically use box when I need things like ssx or event bubbling or any advanced features that would be necessary when composing this component, if it acts purely as a wrapper with no need to bring global in to the css I use div. But I would say it is mostly up to the developers discression, but the I would say that UnicornUI promotes componitization over semantic and global css classes. If I am not extending or taking advantage of the addition props than I don't need to use Box.